spyder_notebook/widgets/main_widget.py

```diff
@@ -178,6 +178,9 @@ def setup(self):
                 section=NotebookMainWidgetOptionsMenuSections.Other,
             )
 
+        # Context menu for notebooks
+        self.tabwidget.actions = [new_notebook_action, open_notebook_action]
```

Comment:
I thought the idea was to add these actions to the context menu of `NotebookWidget`. I mean, it's not clear from the message in our welcome page that you have to right-click on top of the tabwidget to see those actions. Instead, it seems to imply that they will be displayed when right-clicking on that page.

Reply:
@ccordoba12 The idea is to release version 0.4.1 of the plugin with Spyder 5.4.3. That version will have the New/Open Notebook actions in the context menu that is displayed when you right-click anywhere on the page, because that is how it used to be in version 0.3.

Then, version 0.5 of the plugin which will be with Spyder 6 and Jupyter Lab 4 will display the Jupyter context menu when you right-click anywhere on the page, so it will not have the Spyder New/Open Notebook actions. As you say, the message on the welcome page will have to be changed to explain that you need to right-click on the hamburger icon.

At least, that is how I interpreted our discussion, but I am happy to change this.

Reply:
> That version will have the New/Open Notebook actions in the context menu that is displayed when you right-click anywhere on the page, because that is how it used to be in version 0.3.

Ok, sorry for the noise then. I thought those actions would only be displayed when doing a right click on top of the notebooks' tabwidget.

> Then, version 0.5 of the plugin which will be with Spyder 6 and Jupyter Lab 4 will display the Jupyter context menu when you right-click anywhere on the page, so it will not have the Spyder New/Open Notebook actions.

Yeah, I understood that part.

> At least, that is how I interpreted our discussion, but I am happy to change this.

Nop, that's fine.